docs/guides/developing/usage_examples.rst

```diff
@@ -93,15 +93,15 @@ Python
    import PyOpenColorIO as OCIO
    
    try:
-       config = OCIO.GetCurrentConfig()
-       processor = config.getProcessor(OCIO.Constants.ROLE_COMPOSITING_LOG,
-                                       OCIO.Constants.ROLE_SCENE_LINEAR)
-       cpu = processor->getDefaultCPUProcessor()
+      config = OCIO.GetCurrentConfig()
+      processor = config.getProcessor(OCIO.ROLE_COMPOSITING_LOG,
+                                      OCIO.ROLE_SCENE_LINEAR)
+      cpu = processor->getDefaultCPUProcessor()
```

Comment:
Shouldn't it be `cpu = processor.getDefaultCPUProcessor()` ?

Reply:
Good catch.